src/components/section/section.css

```diff
@@ -9,7 +9,7 @@
 }
 
 .section {
-  margin-bottom: var(--spacing-x-large);
```

Comment:
How does this ensure that the last section still have `--spacing-x-large` (as stated in the AC)?

Reply:
Great question. I'll have to revisit this.